src/main/scala/nl/biopet/tools/multigenicsearch/package.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2018 Biopet
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package nl.biopet.tools
+
+package object multigenicsearch {
+  case class SingleVariant(contig: String, pos: Int, samples: List[Boolean])
```

Comment:
/** A single variant site across all samples in the input VCF file
 *
 *  @contig contig names, e.g., chr1
 *  @pos coordinate of a variant, e.g., 17109
 *  @samples a list of Boolean values. True if the corresponding sample has a variant genotype (e.g., 0/1 or 1/1). False if the corresponding sample has a reference genotype. 
 */

Reply:
this is to specific for the class, the boolean meaning is not decided here